Library/Homebrew/dev-cmd/linkage.rb

```diff
@@ -27,6 +33,8 @@ def linkage
       else
         result.display_normal_output
       end
+
+      database_cache.close
```

Comment:
Ideally I'd make the cache usage a block so there doesn't need to be a manual `.close`